UKHomeOffice/sgmr-service
# SPCR-487 Fix radio pre-fill on PC form

### AC / Description of changes
To recreate:
- Create a draft voyage with a PC saved
- Leave voyage plan to dashboard and go back to it
- Click on change PC details
- Add a new PC
- Fill in page 1 and continue to page 2

 Expected behaviour:
- All fields should be blank

 Observed behaviour:
- AIS, MMSI, charterOrHire and VAT questions are pre-filled with details of PC selected in draft

### To Test
- Create a draft voyage with a PC saved
- Leave voyage plan to dashboard and go back to it
- Click on change PC details
- Add a new PC
- Fill in page 1 and continue to page 2
- > All fields should be blank
- Select the new PC
- Complete up until CYA 
- The details of your added vessel should be displayed

### Notes

---

**Remember to merge to feature branches not `master`.**
